config/channel-settings.json

```diff
@@ -734,6 +731,9 @@
         "referenceLocation": {
           "salesOrderBusinessReferences": ["fulfillment.sales_order_business_ref"]
```

Comment:
do you still want this referenceLocation value? seems like it would cause duplicates...